packages/phpstan-rules/build/target-repository/bootstrap.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
not sure, if/when/whether this bootstrap.php script is usefull for the phpstan-rules case. copied it over from `ecs` so we can discuss what needs to be done.